templates/burger-menu-items.twig

```diff
@@ -8,7 +8,7 @@
 		{% set link_ga_action = item.title %}
 	{% endif %}
 	<li class="burger-menu-item nav-item accordion-item {{ item.class }} {{ item == item.current ? 'active' : '' }}">
-		<div class="item-wrapper accordion-header" id="burger-menu-accordion-{{ targetId }}">
+		<div class="burger-menu-item-wrapper accordion-header" id="burger-menu-accordion-{{ targetId }}">
```

Comment:
This is the only markup change. The `item-wrapper` class is too broad to make any sense.

I'll probably do another PR after this one to address the many unsemantic wrapper elements, so might change it again there.